examples/Identifying Local Environments in a Complex Crystal.ipynb

```diff
@@ -0,0 +1,445 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots()\n",
+    "ax.scatter(range(0, env_cluster.num_clusters, 1), num_particles_list)\n",
+    "ax.set_xlabel(\"Cluster keys\")\n",
+    "ax.set_ylabel(\"Number of environment neighbors\")\n",
```

Comment:
I'd recommend a text output rather than a plot here, too. It's only three values, so a scatter plot is a little bit overkill.